src/client/services/server-api/index.js

```diff
@@ -26,7 +26,7 @@ const ServerAPI = {
 
   querySearch(query){
     const queryClone=_.assign({},query);
-    if(/(uniprot:\w+|ncbi:[0-9]+|hgnc:\w+)$/.test(queryClone.q)){queryClone.q=queryClone.q.split(':')[1];}
+    queryClone.q=queryClone.q.replace(/^(uniprot|ncbi|hgnc):/i,"");
```

Comment:
Correct me if I am wrong but won't this just replace the first token?

Reply:
Yes, it will only remove db when the query has format db:id (ncbi:7156 => 7156); otherwise, don't do anything. Maybe I should restrict the condition so that "ncbi:7156 mdm2" won't be replaced.

Should I also make validation of database name (ncbi, uniprot, and hgnc) case sensitive since the links we provide for Unprot should be all lower case?

Reply:
>  Maybe I should restrict the condition so that "ncbi:7156 mdm2" won't be replaced.

Yes.

> Should I also make validation of database name (ncbi, uniprot, and hgnc) case sensitive since the links we provide for Unprot should be all lower case?

No.